### PR TITLE
Remove support for Python2.6

### LINKED ISSUE #116: Drop support for Python 2.6
The `ssl` module in Python 2.6 does not support TLS 1.2. Given that other prominent open source projects, like OpenStack and pyca/cryptography, are moving away from supporting 2.6, PyKMIP should move to deprecate 2.6 support in the next release, ultimately dropping 2.6 support down the road.

### PR DESCRIPTION
**Pull request by PeterHamilton**
This change removes PyKMIP support for Python2.6. The last release of Python2.6 occurred in October 2013, over 3 years ago. It also does not support security features needed to use PyKMIP according to best practices. This change drops tox and TravisCI settings for Python2.6 and updates setup to no longer categorize the library as Python2.6 compliant. The README has been updated accordingly.

Closes #116

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -27,10 +27,10 @@
 setuptools.setup(
     name='PyKMIP',
     version=__version__,
-    description='KMIP v1.1 library',
+    description='KMIP library',
```
Comment (joel-coffman) [inline setup.py]: This change seems unrelated to Python 2.6.

File: setup.py [hunk 2]
```diff
@@ -27,10 +27,10 @@
...
     keywords='KMIP',
     author='Peter Hamilton',
-    author_email='peter.hamilton@jhuapl.edu',
+    author_email='peter.allen.hamilton@gmail.com',
```
Comment (joel-coffman) [inline setup.py]: Ditto

File: setup.py (hunk 1 above)
Comment (PeterHamilton) [inline setup.py]: True, true. Fixed!

File: setup.py (hunk 2 above)
Comment (PeterHamilton) [inline setup.py]: Ditto
